ideascube/configuration/migrations/0002_settings_to_configuration.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+from ideascube.models import Setting
```

Comment:
Ok, so we just weren't talking about the same thing:

* I meant model as in the code that maps a db table into a Python object
* you meant the db table

Indeed, we certainly can remove the db table, and we can do it right away, in the same operation that migrates the existing `Setting`s into `Configuration`s. I'll add that.

> The only thing is indeed to never import models directly in migrations: not only because they could have been deleted

This is what confuses me.

If we remove the model from the code, then whether you import it from the registry or directly, the import will still fail, won't it?